src/components/tutorials/advanced-tutorial.js

```diff
@@ -17,7 +17,7 @@ class AdvancedTutorial extends React.Component {
         <div>
           <div className='temp-grid'>
             <h1 className='ds-h1'>Tutorial: Update Data</h1>
-            <p>In the <a href='/qpp-submissions-docs/tutorial'>first tutorial</a> we covered how to create a measurement set with category performance data, and retrieve the score in three different API requests. This time we're going to build on the previous tutorial look at adding more Quality performance data to the measurement set we created in the previous tutorial (all while running into a problem along the way). All of these examples serve to illustrate how the Submissions API can make it easier to react to and fix issues that arise.</p>
+            <p>In the <a href='/qpp-submissions-docs/tutorial'>first tutorial</a> we covered how to create a measurement set containing Improvement Activities, and retrieve the score in three different API requests. This time we're going to build on the previous tutorial look at adding more Quality performance data to the measurement set we created in the previous tutorial (all while running into a problem along the way). All of these examples serve to illustrate how the Submissions API can make it easier to react to and fix issues that arise.</p>
```

Comment:
ack sorry, missed this. This paragraph should read like:

 <p>In the <a href='/qpp-submissions-docs/tutorial'>first tutorial</a> we covered how to create a measurement set containing Improvement Activities, and retrieve the score in three different API requests. We're now going to build on the previous tutorial by adding another Improvement Activity to the measurement set we created in the previous tutorial (all while running into a problem along the way). All of these examples serve to illustrate how the Submissions API can make it easier to react to and fix issues that arise.</p>